tests/dragonfly/server_family_test.py

```diff
@@ -60,3 +64,23 @@ def test_connection_name(client):
     client.execute_command("CLIENT SETNAME test_conn_name")
     name = client.execute_command("CLIENT GETNAME")
     assert name == "test_conn_name"
+
+'''
+make sure that the scan command is working with python
+'''
+def test_scan(client):
+    def generate_key_val():
+        now = datetime.datetime.now()
+        for i in range(10):
+            string_val = "".join(random.choice(ascii_lowercase) for i in range(random.randint(1, 8)))
+            yield f"key{i}-{time.time()}", f"value={i}{string_val}{now}"
```

Comment:
I have this kind of stuff without timestampts in `utility.py` as `gen_test_data`

I know its just tests, but maybe we can make such utils re-usable

Reply:
OK - I took this test from some offline code I have - I will change it